src/controllers/controlpickermenu.cpp

```diff
@@ -415,6 +415,57 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
         hotcueMainMenu->addMenu(hotcueMoreMenu);
     }
 
+    // Intro/outro range markers
+    QMenu* introOutroMenu = addSubmenu(tr("Intro / Outro Cues"));
+    QString introStartTitle(tr("Intro Start Cue"));
+    QString introEndTitle(tr("Intro End Cue"));
+    QString outroStartTitle(tr("Outro Start Cue"));
+    QString outroEndTitle(tr("Outro End Cue"));
+    QMenu* introStartMenu = addSubmenu(introStartTitle, introOutroMenu);
+    QMenu* introEndMenu = addSubmenu(introEndTitle, introOutroMenu);
+    QMenu* outroStartMenu = addSubmenu(outroStartTitle, introOutroMenu);
+    QMenu* outroEndMenu = addSubmenu(outroEndTitle, introOutroMenu);
+    QList<QMenu*> introOutroSubmenus;
+    introOutroSubmenus.append(introStartMenu);
+    introOutroSubmenus.append(introEndMenu);
+    introOutroSubmenus.append(outroStartMenu);
+    introOutroSubmenus.append(outroEndMenu);
+    QStringList cueTypeTitles;
+    cueTypeTitles.append(introStartTitle);
+    cueTypeTitles.append(introEndTitle);
+    cueTypeTitles.append(outroStartTitle);
+    cueTypeTitles.append(outroEndTitle);
```

Comment:
Have you considered using initializer lists? E.g.:

```suggestion
    QStringList cueTypeTitles = {
        introStartTitle,
        introEndTitle,
        outroStartTitle,
        outroEndTitle,
    }
```

Reply:
I tried that first but ran into errors I was too lazy to resolve. But it was just a syntax issue... Thx! constantly learning
``` c++
    const QStringList cueTypeTitles(
            ...
            );
```
vs.
``` c++
    const QStringList cueTypeTitles = 
            ...
            };
```

Reply:
Ready.